src/classes/HelloPortlandTest.cls

```diff
@@ -0,0 +1,17 @@
+@isTest
+private class HelloPortlandTest {
+	
+	@isTest static void testSayHello() {
+		System.assertEquals('Hello Bob!', HelloPortland.sayHello('Bob'));
+	}
+
+	@isTest static void testBadThing() {
+		Test.startTest();
+		HelloPortland.badThing(2);
+		Test.stopTest();
+
+		List<Account> testAccounts = [Select Id from Account];
```

Comment:
If the table in this query contains more than 50'000 records a runtime exception will be thrown. Consider either introducing a `WHERE` clause, or adding a `LIMIT` to contain the number of records returned.